package.json

```diff
@@ -25,6 +25,7 @@
     "@babel/plugin-transform-runtime": "7.0.0-beta.54",
     "@babel/preset-env": "7.0.0-beta.54",
     "@babel/preset-react": "7.0.0-beta.54",
+    "@babel/runtime": "^7.0.0-beta.54",
```

Comment:
Needed to make it works, there is another PR for that:
https://github.com/embark-framework/embark/pull/656/